### ISSUE TITLE
DeSmuME_0.9.13_x64 [*.bak Files Windows 10 conflict]

### ISSUE THREAD

**Issue by c0untfl0yd** [TASEmulators/desmume#651]
I downloaded the emulator from this github page a few days ago & started making TASes with it. Eh-em so what happens is each time I modify my .dsm file externally a *.bak file is automatically created on my Desktop of all places. 

Long story short after hundreds of .bak files created eventually the Emulator has hijacked the Windows10 registry value of HKEY_CLASSES_ROOT\Folder and has set the Default to "nds_auto_file" instead of "Folder" &therefore I could no longer create New Folders on my windows machine. :) I don't know how to turn off the ".bak" files generation +maybe the emulator needs them to function properly, but anyhow there is no such option in .ini file. I want to keep using the right click New Folder option on my windows machine without having to go to the registry each time to manually change the value.

Please investigate and resolve.

**Comment by zeromus** [comment]
desmume creates the backup movies in the same directory as the .dsm. If you don't want .baks on your desktop, don't put .dsms there. I don't really understand what "modify my .dsm file externally" means. If you edited it in notepad, then desmume has nothing to do with it. Did you modify the .dsm file externally AND THEN ALSO DO SOMETHING ELSE?

Desmume didn't set nds_auto_file. The only work it does on the registry is done by the checkbox "associate nds files" in the path settings dialog, and it doesn't set nds_auto_file. "_auto_file" is set up by explorer when YOU choose to associate nds files when double clicking them.

Maybe you're using some pirate hack version of desmume (perhaps x432r?) We don't support pirate hack versions.

**Comment by c0untfl0yd** [comment]
This is the the version of desmume I am using "[desmume-0.9.13-win64.zip](https://github.com/TASEmulators/desmume/releases/download/release_0_9_13/desmume-0.9.13-win64.zip)"

"Did you modify the .dsm file externally AND THEN ALSO DO SOMETHING ELSE?" - Yes. After I have modified the .dsm externally I load it in desmume again & the movie is played with the new modifications no issues here.

I don't(&didn't) have the "associate nds files" checkbox checked & you'd be correct to assume that it was I who chose to associate .nds files when double clicking them.

I have fixed the registry value anyway & I am not planning to try and recreate the bug somehow since its already fixed.
I'm sure it will not happen again because Desmume didn't set nds_auto_file in the registry.

Thank You.
